### ISSUE TITLE
valid_docs triggers a warning on tuples 

### ISSUE THREAD

**Issue by kevindelord** [realm/SwiftLint#324]
Hi,

A warning is shown by the `valid_docs` rule when a function returns a tuple.
For example: 
```swift
	/**
	Awesome test function

	- returns: super tuple
	*/
	func tupleAsReturnValue() -> (r: CGFloat, g: CGFloat, b: CGFloat) {
		return (r: 1, g: 2, b: 3)
	}
```

There is no warning when the tuple is a parameter.

Thanks a lot for your work !
Regards

**Comment by jpsim** [comment]
Thanks for reporting! We should certainly fix this. Help appreciated! :smile:
